docs/index.md

```diff
@@ -4,7 +4,7 @@ hide_title: true
 id: Docs home
 displayed_sidebar: topLevelSidebar
 toc_max_heading_level: 2
-pagination_next: getting-started/quickstart
+pagination_next: quickstart-managed-scans
```

Comment:
@s-santillan this is causing the Netlify build to fail. This works locally, along with `getting-started/quickstart-managed-scans`. Do you have any ideas as to why this is happening?

Reply:
I'll try something :)